arm-containerregistry/2017-03-01/swagger/containerregistry.json

```diff
@@ -0,0 +1,746 @@
+        "id",
+        "accessKey"
+      ],
+      "properties": {
+        "id": {
+          "description": "The resource ID of the storage account.",
+          "type": "string"
+        },
+        "accessKey": {
+          "description": "The access key to the storage account.",
+          "type": "string"
+        }
+      }
+    },
+    "RegistryUpdateParameters": {
+      "description": "The parameters for updating a container registry.",
+      "properties": {
+        "tags": {
+          "description": "The tags for the container registry.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "$ref": "#/definitions/RegistryPropertiesUpdateParameters",
+          "description": "The properties that the container registry will be updated with.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "RegistryPropertiesUpdateParameters": {
+      "description": "The parameters for updating the properties of a container registry.",
+      "properties": {
+        "adminUserEnabled": {
+          "description": "The value that indicates whether the admin user is enabled. This value is false by default.",
+          "type": "boolean"
+        },
+        "storageAccount": {
+          "$ref": "#/definitions/StorageAccountParameters",
+          "description": "The parameters of a storage account for the container registry. If specified, the storage account must be in the same physical location as the container registry."
+        }
+      }
+    },
+    "RegistryListResult": {
+      "description": "The result of a request to list container registries.",
+      "properties": {
+        "value": {
+          "description": "The list of container registries. Since this list may be incomplete, the nextLink field should be used to request the next list of container registries.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Registry"
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next list of container registries.",
+          "type": "string"
+        }
+      }
+    },
+    "RegistryListCredentialsResult": {
+      "description": "The response from the ListCredentials operation.",
+      "properties": {
+        "username": {
+          "description": "The username for a container registry.",
+          "type": "string"
+        },
+        "passwords": {
+          "description": "The list of passwords for a container registry.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegistryPassword"
+          }
+        }
+      }
+    },
+    "RegistryPassword": {
+      "description": "The login password for the container registry.",
+      "properties": {
+        "name": {
+          "description": "The password name.",
+          "enum": [
+            "password",
```

Comment:
Minor - To me password and password2 does not sound that great. can we rename password to password1 as well, so that pair becomes password1 and password2